firefox_puppeteer/base.py

```diff
@@ -22,3 +23,21 @@ def marionette(self):
 
     def get_marionette(self):
         return self.marionette
+
+
+class UIBaseLib(BaseLib):
+    """A base class for all UI element wrapper classes inside a chrome window."""
+
+    def __init__(self, marionette_getter, window):
+        BaseLib.__init__(self, marionette_getter)
+
+        # importing globally doesn't work
+        from .ui.windows import BaseWindow
```

Comment:
Local import statements have global side effects, so I'd rather omit this check (or convert to something like a comparison against type(window).__name__) and file a follow up to get this figured out.